src/pykeen/evaluation/__init__.py

```diff
@@ -1,6 +1,71 @@
 # -*- coding: utf-8 -*-
 
-"""Evaluation."""
+r"""
+Knowledge graph embeddings models are usually evaluated based on link prediction, which is defined
+as predicting the tail/head entities for $(h,r)$/$(r,t)$ pairs.
+
+However, given the fact that both the training and the test set usually only contain true facts, negative samples are
+generated by corrupting these positive triples. For each test triple $t^{+} = (h,r,t) \in \mathcal{K}_{test}$ two sets
+of corrupted triples are constructed:
+
+    * $\mathcal{H}(r, t) = \{(h', r, t) \mid h' \in \mathcal{E} - \{h\}$
+    * $\mathcal{T}(h, r) = \{(h, r, t') \mid t' \in \mathcal{E} - \{t\}\}$
+
+where $\mathcal{H}(r, t)$ contains all the triples where the head entity has been corrupted, and $\mathcal{T}(h, r)$
+contains all the triples with corrupted tail entities.
+
+Among the corrupted triples in $\mathcal{H}(r, t)$ / $\mathcal{T}(h, r)$, there might be true triples that are part of
+the knowledge graph.
+If these false negatives are ranked higher than the current test triple $t^{+}$, the results might get distorted.
+Therefore, the \emph{filtered} evaluation setting has been proposed \cite{Bordes2013}, in which the corrupted triples
+are filtered to exclude known true facts from the train, validation and test set. Thus, the rank does not decrease when
+ranking another true entity higher.
+
```

Comment:
use section headers with the `------` level to break this up into a "background/normal usage" section and then a more specific one that's addressing the issue raised in the PR (you can also link back to the PR from the docs for extra context)

Reply:
Seems to be exceeded by a13aa1770e3d6238da082474606bd77f1f14c1cf and 29c89e4dbdfbddd0dec91b03fe5c21b7b2e4501c.

Reply:
Not quite yet - the text that remains still needs to be organized into sections :)